src/registrar/views/domain_request.py

```diff
@@ -116,7 +116,7 @@ class DomainRequestWizard(TemplateView):
         Step.ABOUT_YOUR_ORGANIZATION: lambda self: self.domain_request.about_your_organization is not None,
         Step.SENIOR_OFFICIAL: lambda self: self.domain_request.senior_official is not None,
         Step.CURRENT_SITES: lambda self: (
-            self.domain_request.current_websites.exists() or self.domain_request.requested_domain is not None
+            self.domain_request.current_websites.exists() or self.domain_request.senior_official is not None
```

Comment:
Technically we could remove the first conditional, but leaving it since it's an accurate condition.